app/views/layouts/application.html.haml

```diff
@@ -1,18 +1,17 @@
-!!!
```

Comment:
This is the doctype of the document. So this line would be required. https://www.w3schools.com/tags/tag_doctype.asp